
# The main pipelines for metabarcoding separated by choice of marker genes.

### The most used marker genes and its applications:

- **16S rRNA gene** 
	- ~1600 bp and 9 Hypervariable regions
			V2-V3
			V3-V4
			V4
- **12S rRNA gene**
- **18S rRNA gene**
- **ITS spacer region** 
		- ITS1
		- ITS2
- **COI gene**
		- 
> 

### Data exploration systems and databases:

https://tools.biodiversitydata.se/
https://lists.biodiversitydata.se/public/speciesLists?lang=en-US

https://asv-portal.biodiversitydata.se/?lang=en-US

BOLD
MGNIFY


### Taxonomy assignment algorithms
Tiara> Deep learning/based classification system for euk sequences
https://academic.oup.com/bioinformatics/article/38/2/344/6375939







### References:
Overview of barcode markers and major reference libraries:
https://link.springer.com/article/10.1007/s12686-022-01291-2


16S rRNA gene  region choice:
https://www.nature.com/articles/sdata20197#:~:text=The%2016S%20rRNA%20gene%20is,help%20identify%20genus%20or%20species.
ITS spacer region: https://www.pnas.org/doi/10.1073/pnas.1117018109

COI - Cytochrome c oxidase subunit 1 gene
https://parasitesandvectors.biomedcentral.com/articles/10.1186/s13071-017-2457-1
Putting COI Metabarcoding in Context: The Utility of Exact Sequence Variants (ESVs) in Biodiversity Analysis
https://www.frontiersin.org/articles/10.3389/fevo.2020.00248/full

Mitochondria genome:
https://www.nature.com/scitable/topicpage/mtdna-and-mitochondrial-diseases-903/#:~:text=The%20mitochondrial%20genome%20contains%2037,the%20powerhouses%20of%20our%20cells.

GBIF eDNA reference
https://docs.gbif.org/publishing-dna-derived-data/1.0/en/
